asmcli/commands/validate.sh

```diff
@@ -55,6 +55,8 @@ validate_dependencies() {
         enable_service_mesh_feature
       fi
       if [[ "${CA}" == "managed_cas" ]]; then
+        info "Wait 3 minutes for GKE HUB API enablement to propagate to the systems ..."
+        sleep 180
```

Comment:
should this come after x_enable_workload_certificate_on_fleet?

Reply:
This should be before x_enable_workload_certificate_on_fleet because before enabling workload certificate on fleet, the GKE HUB API enablement needs to be propagated to the systems.